### DISCUSSION TITLE
What's the difference between DB4 SQLCipher.exe and DB4 SQLite.exe?

### DISCUSSION THREAD

**Discussion by victor-ajayi** [sqlitebrowser/sqlitebrowser#3049] [Q&A] [answered]
They both seem to open the same app, I'm not sure what the difference between the two is

**Comment by justinclift** [comment, marked as answer]
The `SQLCipher` one has built in encryption, based on [SQLCipher](http://sqlcipher.net).  The SQLCipher library gets released every now and then, but is generally based upon a slightly older version of SQLite than *whatever the latest release is*.

The other one doesn't have encryption built in.  Instead, it uses the very latest version of SQLite that was available at the time of release.

Does that help? :smile:

**Reply by victor-ajayi** [reply]
Yes, it does. Plenty thanks!
